deepcell_label/models.py

```diff
@@ -454,6 +456,19 @@ def get_labeled_array(self, feature, frame):
         label_arr = label_frame.frame[..., feature]
         return add_outlines(label_arr)
 
+    def add_channel(self, channel):
+        """
+        Adds a channel to an existing project.
+        """
+        new_raw_frames = []
+        for i, raw_frame in enumerate(self.raw_frames):
+            old_array = raw_frame.frame
+            new_array = np.concatenate((old_array, channel[i]), axis=-1)
+            new_raw_frames += [RawFrame(i, new_array)]
```

Comment:
I think the `.append` method is much faster/lighter. The concatenate operation ends up creating additional lists.